indra/db/database_manager.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import absolute_import, print_function, unicode_literals
 from builtins import dict, str
+from sqlalchemy.orm.attributes import InstrumentedAttribute
```

Comment:
When adding a new import path like this, it often breaks the documentation build (if the parent module is mocked), please add `sqlalchemy.orm` to the mocks list in doc/conf.py and test that the build works.

Reply:
`sqlalchemy.orm` is already included in the mock modules. This isn't the first time it's been included. It's used extensively in the content manager.

Reply:
It's `sqlalchemy.org.attributes` then, either way it's broken.

Reply:
Done